.github/workflows/development_javadoc.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Development Javadoc
+
+on:
+  push:
+    branches:
+     - 'master'
```

Comment:
Do you want to always do it? This might mean there is documentation out ahead of a release? Would building on tags be better?

Reply:
yeah, we have documentation for development versions also in addition to release.